code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -1359,6 +1359,63 @@
 /obj/item/weapon/reagent_containers/food/drinks/shaker/reaction
 	flags = FPRINT  | OPENCONTAINER | SILENTCONTAINER
 
+
+//bluespace shaker
+
+/obj/item/weapon/reagent_containers/food/drinks/bluespaceshaker
+	name = "\improper bluespace shaker"
+	desc = "A bluespace shaker to mix drinks in."
+	icon_state = "bluespaceshaker"
+	origin_tech = Tc_BLUESPACE + "=4;" + Tc_MATERIALS + "=6"
+	starting_materials = list(MAT_GLASS = 2000)
+	w_type = RECYK_GLASS
+	amount_per_transfer_from_this = 10
+	volume = 300
```

Comment:
?

Reply:
starting_materials: the amount is identical to bluespace beakers (what i originally based the material cost on). this variable is the amount of materials you get for directly inserting the item into a protolathe.
w_type: the type of material that it gets recycled into in the waste area of the station

![image](https://github.com/vgstation-coders/vgstation13/assets/11482753/081622e9-32c7-4443-a1fe-594bb38bee82)
![educate](https://github.com/vgstation-coders/vgstation13/assets/11482753/61676f71-d3c3-4ee8-9b6d-4e03ad6782fe)

Reply:
will update this to more accurately reflect the cost now that i changed it, i had forgotten about this

Reply:
your OP stuff says 200u, but the code says 300u

Reply:
?